.eslintrc

```diff
@@ -1,42 +1,8 @@
 {
-  "extends": ["airbnb", "prettier"],
-  "env": {
-    "es6": true,
-    "browser": true,
-    "node": true,
-    "mocha": true,
-    "jasmine": true,
-    "protractor": true
-  },
-  "parser": "babel-eslint",
-  "rules": {
-    "prettier/prettier": [
-      "error",
-      { "trailingComma": "all", "singleQuote": true }
-    ],
-    "jsx-a11y/anchor-is-valid": [ "error", { "components": [ "Link" ], "specialLink": [ "to" ] } ],
-    "import/prefer-default-export": "off",
-    "react/prefer-stateless-function": 0,
-    "import/no-unresolved": 1,
-    "import/extensions": 0,
-    "import/no-extraneous-dependencies": 0,
```

Comment:
"import/no-extraneous-dependencies": 0 This is required also, it used to complain with Volto itself or Volto dependencies. Maybe, airbnb forced, and react-app doesn't, then it's not required.